### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by SelmaNdapanda**
In this pull request:

- I created the desktop version of my portfolio

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -743,3 +747,677 @@ button:active {
...
+    position: fixed;
+    width: 100%;
+    z-index: 1;
+    display: flex;
+    flex: 1 1 auto;
+    text-align: center;
+    padding: 1em;
+    background-color: #fff;
+    justify-content: space-between;
+  }
```
Comment (zieeco) [inline style.css]: - Please, kindly increase the padding top to make your design close to that of the Figma design as much as you can

File: style.css [hunk 2]
```diff
@@ -743,3 +747,677 @@ button:active {
...
+    flex-direction: row;
+    gap: 24px;
+    align-items: center;
+    box-sizing: border-box;
+    width: 100%;
+    height: 924px;
+    background: url("images/abt-myself-desktop.png") no-repeat;
+    border-top-right-radius: 80px;
+    top: 3600px;
+  }
```
Comment (zieeco) [inline style.css]: - Please, kindly add a `background-size: cover` to fix the issue with the about section not taking the full width as seen below in your design:

spacee_behind

File: style.css [hunk 3]
```diff
@@ -743,3 +747,677 @@ button:active {
...
+    background-size: contain;
+    display: flex;
+    flex-direction: column;
+    width: 99vmax;
+    height: 846px;
+    align-items: center;
+    margin: 0 auto;
+    text-align: center;
+    justify-content: center;
+  }
```
Comment (zieeco) [inline style.css]: - Please, kindly ensure the background image for the desktop version is showing. please fix this issue.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @SelmaNdapanda,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (SelmaNdapanda) [reply style.css]: Thank you, I will solve it right away.

File: style.css (hunk 2 above)
Comment (SelmaNdapanda) [reply style.css]: Thank you for the suggestion, it is resolved

File: style.css (hunk 3 above)
Comment (SelmaNdapanda) [reply style.css]: It is now showing. thank you very much

File: style.css [hunk 4]
```diff
@@ -743,3 +747,680 @@ button:active {
   background: #0416b6;
   color: white;
 }
+
+@media screen and (min-width: 768px) {
```
Comment (jasonfelice) [inline style.css:751]: - [ ] **[Optional]** I highly recommend fixing this (refer to the image attached) grey space.

image

**Review by jasonfelice — APPROVED** [review]
Hi @SelmaNdapanda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
No Linter Errors✔️
Good commit messages✔️
Descriptive PR✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**